database/postgresql/build.gradle.kts

```diff
@@ -13,12 +13,10 @@ dependencies {
     implementation(project(":model"))
     implementation(project(":commons"))
 
-    implementation("io.quarkus:quarkus-reactive-pg-client")
-    implementation("org.postgresql:postgresql:42.6.0")
-    implementation("io.smallrye.reactive:mutiny")
+    implementation("io.quarkus:quarkus-hibernate-orm")
+    implementation("io.quarkus:quarkus-jdbc-postgresql")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.15.2")
```

Comment:
Let's add this version to the `gradle/versions` or to `bom/gradlebuild.kts`. It would help manage versions better.

Reply:
strongly agree, forgot to fix it ;)